Floofbot/Modules/Administration.cs

```diff
@@ -84,6 +78,13 @@ public class Administration : InteractiveBase
             builder.AddField("Moderator", $"{Context.User.Username}#{Context.User.Discriminator}");
 
             await Context.Channel.SendMessageAsync("", false, builder.Build());
+
+            //sends message to user
+            builder.Title = "⚖️ Ban Notification";
+            builder.Description = $"You have been banned from {Context.Guild.Name}";
+            builder.AddField("Reason", reason);
+            builder.Color = ADMIN_COLOR;
+            await badUser.SendMessageAsync("", false, builder.Build());
```

Comment:
This embed would still have the fields added above (`User ID` and `Moderator`), no?

For safety, I'd suggest using `new EmbedBuilder` again for the ban embed.